multiload/global.h

```diff
@@ -81,6 +75,17 @@ struct _MultiloadApplet
 	GtkWidget *prop_dialog;
 	GtkWidget *notebook;
 	int last_clicked;
+
+    long cpu_time [NCPUSTATES];
+    long cpu_last [NCPUSTATES];
+    int cpu_initialized;
+
+    double loadavg1;
+    NetSpeed *netspeed_in;
+    NetSpeed *netspeed_out;
+    guint net_threshold1;
+    guint net_threshold2;
+    guint net_threshold3;
```

Comment:
the indentation is wrong

Reply:
No it isn't

Reply:
take a look at _LoadGraph

Reply:
It looks like I had a deja vu, didn't we talk about it recently?

Reply:
seems do you like to see a weird file with misaligned indentations, I don't agree

Reply:
I formatted the rest of the file in a separate commit, I think there is no other PR that modifies the formatted code ;)

Reply:
Thanks!